css/css-break/reference/widows-orphans-001-ref.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Fragmentation level 3 Test Reference File</title>
+<link rel="author" title="Florian Rivoal" href="https://florian.rivoal.net/">
+<style>
+div {
+  position: absolute;
+  columns: 2;
+  column-fill: balance-all;
```

Comment:
* If you mean to make that change on the ref only, sure, but why? Both approaches should give the same result. I don't object to changing, but I don't see why it's an improvement.

* If you mean to make that change to the tests themselves as well, that would break how these tests work. Different tests can certainly be written with fixed column sizes, but that's another story. Just because different tests can be written doesn't mean these are wrong..